boards/mro/cannode/init/rc.board_sensors

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+#
+# board sensors init
+#------------------------------------------------------------------------------
+rm3100 -s start
+
+vl53l1x -I start
```

Comment:
Respect the parameter SENS_EN_VL53L1X?

Also, the i2c should be external (-X).

Reply:
Ahh, I had -X before but second guessed myself. It's all I have to test right now since I am waiting for tfmini and airspeed sensor.